core/src/main/java/com/gentics/mesh/cache/TotalsCacheImpl.java

```diff
@@ -0,0 +1,145 @@
+package com.gentics.mesh.cache;
```

Comment:
Owa. Dev branch still needs upmerges from hotfix-2.0.

Reply:
Unfortunately, I won't opt for getting away all the recent hotfix-2.0 changes from this branch - it will be a pain to merge it back afterwards, with no context knowledge.